nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -35,6 +35,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - `services.avahi.nssmdns` got split into `services.avahi.nssmdns4` and `services.avahi.nssmdns6` which enable the mDNS NSS switch for IPv4 and IPv6 respectively.
   Since most mDNS responders only register IPv4 addresses, most users want to keep the IPv6 support disabled to avoid long timeouts.
 
+- `xfsprogs` was updated to version 6.5.0, which enables reverse mapping (rmapbt) and large extent counts (nrext64) by default.
+   Support for these features was added in kernel 4.9 and 5.19 and nrext64 was deemed stable in kernel 6.5.
+   **Current GRUB2 cannot boot from these filesystems**.
```

Comment:
cc @K900, since you last updated the grub2 package.

Can you either remove this if the next update supports this (or just ping me and I'll take a look)?

Reply:
I think it was fixed in GRUB master, but our revision doesn't have it yet.

Reply:
Right. So my suggestion would be to merge this as is and update this release-notes entry once (our) grub2 can read these filesystems.

But I'm obviously open to other suggestions. IMO booting from XFS is quite a nice use-case.

Reply:
https://lists.gnu.org/archive/html/grub-devel/2023-10/msg00168.html was released as part of grub 2.12, which we have since #275699